.env.example

```
@@ -45,3 +45,4 @@ SECRET_KEY=change-me-in-production
 # YouTube API Configuration
 # Get your API key from: https://console.cloud.google.com/
 YOUTUBE_API_KEY=YOUR_API_KEY
+OFFLINE_MODE=True
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OFFLINE_MODE key should go before the YOUTUBE_API_KEY key

(UnorderedKey)